src/makeCalculator.js

```diff
@@ -37,7 +37,42 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add(par) {
+      calculator.result += par;
+
+      return calculator.result;
+    },
+    subtract(par) {
+      calculator.result -= par;
```

Comment:
Use 'this' keyword instead of referring to 'calculator' directly to avoid hardcoding the object name